src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/HPackEncoder.cs

```diff
@@ -30,7 +30,7 @@ public static bool EncodeIndexedHeaderField(int index, Span<byte> destination, o
             // | 1 |        Index (7+)         |
             // +---+---------------------------+
 
-            if (destination.Length != 0)
+            if (!destination.IsEmpty)
```

Comment:
This one reduces the assembly size, see [here](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8GuGbunHgECABsGAWQAUAZQAO2AHYAeYAE8MMAHwMAJjFwYAlvOyGI8tAwgBXDAyO3V63AHUoBjOvkBKXt3a++LgMAMwYJAEJdfSMTAzMAOgBJXABRfBkMFR9qQL8A3O4ow2NTeQBtAAYAXQYAXgYKhAAOCoBufIKGRz1Xd086hnJ2nM7uYgB2BgwoKxhhzoBfDu5lrm6XNw8YeQG21aZJ4OxhXDn8pZGufPouoVEAOWk5JW6tIpjSi2tbey61Hs2nmyfH8l0CITC7xKcXk8QAMtsAOYYAAWDHC9QqwIKoNGhT0xViZkqNUxzT2YM6616Wx29SG+z4EymMzOlO4F06+2pgO2u3mBWZRxObL4FwWQA=).